pkg/ccl/importccl/testdata/mysqldump/db.sql

```diff
@@ -63,55 +63,55 @@ LOCK TABLES `everything` WRITE;
 UNLOCK TABLES;
 
 --
--- Table structure for table `simple`
+-- Table structure for table `second`
```

Comment:
This file is changing without an accompanying change to its generation script. Did you miss that in this commit?

Reply:
no, it was hand-edited before to re-order the tables, until this change went in to remove that limitation.